meinberlin/apps/cms/templates/meinberlin_cms/includes/storefronts.html

```diff
@@ -32,7 +32,13 @@ <h2 class="storefront__proj-text {% if item.project.name|length > 95 %} storefro
         <div class="status-bar__active">
             <span class="status-bar__active-fill" style="width:{{ item.project.active_phase_progress }}%"></span>
         </div>
-        <span class="status-bar__status"><i class="fas fa-clock"></i>{% blocktrans with time_left=item.project.time_left %} {{ time_left }} remaining {% endblocktrans %}</span>
+        <span class="status-bar__status"><i class="fas fa-clock"></i>
+          {% if (item.project.time_left < 365) %}
+          {% blocktrans with time_left=item.project.time_left %} {{ time_left }} remaining {% endblocktrans %}
+          {% else %}
+          <span>boo</span>
+          {% endif %}
+        </span>
     </div>
```

Comment:
@MagdaN this was my first attempt at adding the logic to the django, i removed it in the next commit but was just wondering if I was even vaguely on the right track?